core/src/main/java/cz/xtf/core/openshift/CLIUtils.java

```diff
@@ -17,14 +19,44 @@
 @Slf4j
 public class CLIUtils {
```

Comment:
We will need some proof changes are good. Either provide test run (EapDockerfileTest) of OS TS which is using this code. Or introduce CLIUtilsTest.java (generated by AI code assistant)